scripts/utils/generate_ts.sh

```diff
@@ -27,6 +27,17 @@ printn Y "Generating strings... "
 python3 cache/generate_strings.py src/translations/strings.yaml -o translations/generated
 print G "done."
 
+# Helper method - scan a directory and generate a QRC file which includes all QML files.
```

Comment:
Note: CMake will actually generate generate this file if we just build the `mozillavpn-ui` module.